README.md

```diff
@@ -91,7 +91,7 @@ There exist build rules for several systems:
 * **Conan**: `conanfile.py` in source root
 * **PlatformIO**: https://platformio.org/lib/show/431/Nanopb
 * **PyPI/pip**: https://pypi.org/project/nanopb/
-* **vcpkg**: https://vcpkg.info/port/nanopb
+* **vcpkg**: https://vcpkg.io/en/package/nanopb/
```

Comment:
I think the last slash is incorrect? At least it seems to give 404 to me.

Reply:
Ahh yeah, the final slash is incorrect:
https://vcpkg.io/en/package/nanopb should work